Calculator/CalculatorTests/CalculatorTests.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by 조용현 on 2023/01/17.
+//
+
+import XCTest
+@testable import Calculator
+
+final class CalculatorTests: XCTestCase {
+    
+    var sut: LinkedList!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        sut = LinkedList()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        sut = nil
+
+    }
```

Comment:
서식을 일관화하기 위해 빈칸 한줄을 없애야겠습니다 :)
```suggestion
    override func tearDownWithError() throws {
        try super.tearDownWithError()
        sut = nil
    }
```

Reply:
불필요한 개행 처리에 더욱 신경 쓰겠습니다! 👍🏻